src/act.cabal

```diff
@@ -23,8 +23,11 @@ common deps
                  bytestring       >= 0.10.8,
                  sbv              >= 8.4,
                  mtl              >= 2.2.2,
-                 utf8-string      >= 1.0.1.1
-  other-modules: Lex ErrM Extract Parse RefinedAst K HEVM Coq Syntax Type Prove Print Enrich
+                 utf8-string      >= 1.0.1.1,
+                 template-haskell >= 2.14,
```

Comment:
where is template haskell used?

Reply:
[When deriving instances](https://github.com/ethereum/act/blob/f7715c6cd40db9317a7ddbb71b4baf02ca226bc5/src/RefinedAst.hs#L167) of `HFunctor`, `HFoldable`, `ShowHF`, `EqHF` and creating some "constructors" (really functions) which allow us to create `Exp a`s as before.